measures/2019/measures-data.json

```diff
@@ -47936,7 +47936,7 @@
     "strata": [
       {
         "name": "delayed",
-        "description": "Advance 3B: Procedure delayed from scheduled start time"
+        "description": "Advance 3B: Procedure delayed from scheduled start time;"
       },
```

Comment:
The semicolon, weird, right? Also intended?

Reply:
I doubt it's intended but I feel like it should be fixed by whoever gives us the csv instead of by us. What do you think @sarahnw ?

Reply:
I don't think it matters. QCDR measures aren't shown in the shopping cart. These are how the measures are delivered. If we think it's that important we can give them editing feedback b/c it's displayed potentially elsewhere, but in the grand scheme of things I don't think it matters.